site/public/js/forum.js

```diff
@@ -160,12 +160,26 @@ function publishFormWithAttachments(form, test_category, error_message) {
     return false;
 }
 
-function createThread() {
-    return publishFormWithAttachments($(this), true, "Something went wrong while creating thread. Please try again.");
+function createThread(e) {
+    e.preventDefault();
+    try {
+        return publishFormWithAttachments($(this), true, "Something went wrong while creating thread. Please try again.");
+    }
+    catch{
+        alert("There was an irrecoverable fault. Kindly contact administrator.");
```

Comment:
Please change the error message to "Something went wrong. Please try again"

In this case if creating a thread/post fails an instructor wouldn't be able to do anything anyways and the student can always try again

Reply:
Done!